CAIPs/caip-x.md

```diff
@@ -0,0 +1,320 @@
+---
+caip: x
+title: JSON-RPC Provider Lifecycle Methods for Session Management with CAIP-25 Sessions
+author: [Alex Donesky] (@adonesky1)
+discussions-to: TBD
+status: Draft
+type: Standard
+created: 2024-06-07
+requires: 25, 217
+---
+
+## Simple Summary
+
+CAIP-285 introduces new "lifecycle" methods and a new event for managing authorizations within an existing CAIP-25 session, allowing for addition, revocation, and retrieval of authorizations. These methods provide an alternative to session management via `sessionId`s, allowing `sessionId`s to be optional for CAIP-25.
```

Comment:
This file will be removed. Leaving it here for additional context for the time being

Reply:
Counterargument: what if you left it up as an Implementer's Guide that explains the equivalence between legacy sessionId-managed sessions and transport-secured/single-session sessions?